### ISSUE TITLE
Add Activation more than RELU

### ISSUE THREAD

**Issue by kayabaakihiko13** [eduardoleao052/js-torch#14]
Within the concept of activation,there are many algorithms used,not just ReLU, There are numerous concepts as outlined below in the following link.

i have refence that from this [activation](https://www.tensorflow.org/api_docs/python/tf/keras/activations)

**Comment by eduardoleao052** [comment]
Absolutely, I think you are right! I made the repository focused on NLP, which uses mainly the ReLU and Softmax activation functions. However, I should definitely add more to increase the reach of the library.

**Comment by kayabaakihiko13** [comment]
> Absolutely, I think you are right! I made the repository focused on NLP, which uses mainly the ReLU and Softmax activation functions. However, I should definitely add more to increase the reach of the library.

i with you, because after read your file `layers.ts`,i have idea to adding other activation algorithm
